app/assets/javascripts/app/views/header_view.js

```diff
@@ -7,36 +7,27 @@ app.views.Header = app.views.Base.extend({
   className : "dark-header",
 
   events : {
-    "click ul.dropdown li:first-child" : "toggleDropdown",
+    "click ul.nav.navbar-nav li:first-child" : "toggleDropdown",
     "focusin #q": "toggleSearchActive",
     "focusout #q": "toggleSearchActive"
   },
 
   initialize : function(options) {
-    $(document.body).click($.proxy(this.hideDropdown, this));
     return this;
   },
 
   menuElement : function() {
-    return this.$("ul.dropdown");
+    return this.$("ul.nav.navbar-nav");
```

Comment:
This would be an excelent place of an ID. ID are wonderful for attaching JS events handlers and functions. The mark up will definitely change later, as we move to BS3.